community/community_support_process.md

```diff
@@ -0,0 +1,25 @@
+# Community Support Process
+
+## Monthly Community Tracker
+
+**Active tracker** -- [2021 - April Community Tracker](https://github.com/orgs/openebs/projects/31)
+
+The OpenEBS maintainers track community support issues using the monthly Community Tracker projects. The goal of the tracker is to lubricate the support process and gather feedback. It is expected that this project will lead to faster resolution and make OpenEBS user adoption easier.
```

Comment:
OpenEBS uses GitHub Projects to track the slack support tasks. A new tracker is created for each month. The monthly community trackers can be found [here](https://github.com/orgs/openebs/projects/).

The trackers are used to ensure that feedback from the community is available in a format that can be consumed by contributors and maintainers.

Reply:
@niladrih - can we replace the above two sections with the content provided. Giving links to a specific month will make the doc outdated, so prefer to link to the root project.